packages/flutter_test/lib/src/test_default_binary_messenger.dart

```diff
@@ -116,11 +121,17 @@ class TestDefaultBinaryMessenger extends BinaryMessenger {
   // can implement the [checkMockMessageHandler] method.
   final Map<String, Object> _outboundHandlerIdentities = <String, Object>{};
 
+  /// Handler that intercepts and responds to outgoing messages, pretending
```

Comment:
This doc needs a little more details describing how to use this and how setting this affects how the regular handler gets called.

Reply:
There should also be a note on setMockMessageHandler linking back to this.

And this doc should probably link to setMockMessageHandler as well.

Reply:
Some of the information included at https://master-api.flutter.dev/flutter/flutter_test/TestDefaultBinaryMessenger/setMockMessageHandler.html also belongs here, I believe.

Reply:
Thanks for the review! :)
Added some more documentation, WDYT?